common/common.go

```diff
@@ -0,0 +1,26 @@
+package common
+
+import (
+	c "github.com/codilime/floodgate/config"
+	"github.com/codilime/floodgate/gateclient"
+	"github.com/codilime/floodgate/parser"
+	rm "github.com/codilime/floodgate/resourcemanager"
+)
+
+// GetResourceManager return a pointer to a initialized ResourceManager instance.
+func GetResourceManager(configPath string) (*rm.ResourceManager, error) {
```

Comment:
I am not convinced if keeping two ways of creating ResourceManager is a good idea. We already have one way to get ResourceManager:
```go
rm := &ResourceManager{}
rm.Init(...)
```
So maybe we should split that ResourceManager you proposed into two entities:
1. Something like old `Sync`, but without keeping resources inside it. It would be responsible for fetching resources’ remote state from spinnaker and sending local resources’ state to Spinnaker.
2. Parent entity – ResourceManager – that would be created with config and use Parser to load local resources’ state and Sync to load remote resources’ state and propagate changes.

Alternative would be to conflate both `GetResourceManager` and `ResourceManager.Init`.

Reply:
Conflating both seems like the best option for now from my point of view.